test/http.spec.js

```diff
@@ -0,0 +1,29 @@
+import nock from 'nock';
+import path from 'path';
+import downloadFile from '../src/download-file';
+import fs from 'fs';
+import expect from 'expect';
+
+nock('http://local.foo.com')
```

Comment:
Disclaimer: I'm not familiar with `nock`. Is this the only configuration we will ever need?
If not we can create a couple of helpers that returns a nock instance with different configurations.

`helper.nockServerOk(...)` - `helper.nockServerNotOk(...)` etc...